addons/stock/report/report_stockpicking_operations.xml

```diff
@@ -94,6 +94,12 @@
                                                 <span t-if="o.state != 'done'" t-field="ml.reserved_uom_qty"/>
                                                 <span t-if="o.state == 'done'" t-field="ml.qty_done"/>
                                                 <span t-field="ml.product_uom_id" groups="uom.group_uom"/>
+                                                <t t-if="ml.move_id.product_packaging_id">
+                                                    <span> (</span>
+                                                    <span t-field="ml.move_id.product_packaging_qty"/>
+                                                    <span t-field="ml.move_id.product_packaging_id"/>
+                                                    <span>)</span>
+                                                </t>
```

Comment:
This will most likely cause issues because a move can have multiple move lines (e.g. different lots are used to fulfill one move or they are placed in different packs).

Reply:
I am not sure I fully understand this. So what you mean is a move can split into different move lines so for each line I need to get the quantity number of the move line and its packaging instead of the move? Then how would you suggest a fix in this case. I thought the fields product_packaging_qty and product_packaging_id are in stock move so I guess I just put them in.

Reply:
Commit: bdf7da8

- Created fields and methods to compute the packaging in move_line. 
- Also, resolved the disappearing packaging in the delivery slip at the done state. The problem was we also try to merge move lines in _get_aggregated_product_quantities. This created a whole lot of problems since now we have no fixed "quantity". So my solution was to create a compute_product_packaging_qty method that accepts a quantity argument so we can process the packaging before returning so we have the final quantities in compute_packaging (a method inside _get_aggregated_product_quantities).
- Actually, maybe further testing is required (the India testing step) since this was a big change